cheatsheets/Vim.rb

```diff
@@ -209,6 +209,37 @@
             command '{Visual}~'
             name 'Switch case of highlighted text'
         end
+        entry do
+            command '{Visual}U'
+            name 'Uppercase highlighted text'
+    end
+        entry do
+              command 'SHIFT+I<comment-char>ESC'
+              name 'Comment Vertical Selection'
```

Comment:
Please follow the [Guidelines](https://github.com/Kapeli/cheatsheets#guidelines). No title case in entry names.